.golangci.yaml.in

```diff
@@ -0,0 +1,204 @@
+version: "2"
+run:
+  concurrency: 4
+linters:
+  enable:
+    - ginkgolinter
+    - importas
+    - logcheck
+    - misspell
+    - nilerr
+    - revive
+    - unconvert
+    - unparam
+    - whitespace
+  disable:
+    - unused
+  settings:
+    importas:
+      alias:
+        # External imported packages
+        - pkg: k8s.io/api/(\w+)/(v[\w\d]+)
+          alias: $1$2
+        - pkg: k8s.io/apimachinery/pkg/apis/(\w+)/(v[\w\d]+)
+          alias: $1$2
+        - pkg: k8s.io/apimachinery/pkg/api/([^m]\w+)
+          alias: api${1}
+        - pkg: k8s.io/apimachinery/pkg/util/(\w+)
+          alias: util${1}
+        - pkg: k8s.io/autoscaler/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1
+          alias: vpaautoscalingv1
+        - pkg: k8s.io/client-go/tools/clientcmd/api/(\w+)
+          alias: clientcmd${1}
+        - pkg: k8s.io/component-base/config
+          alias: componentbaseconfig
+        - pkg: k8s.io/component-base/config/([\w\d]+)
+          alias: componentbaseconfig$1
+        - pkg: sigs.k8s.io/controller-runtime/pkg/client/fake
+          alias: fakeclient
+        - pkg: sigs.k8s.io/controller-runtime/pkg/log
+          alias: logf
+        - pkg: go.uber.org/mock/gomock
+          alias: gmock
+        # Gardener extension package
+        - pkg: github.com/gardener/gardener/extensions/.*/(\w+)/mock$
+          alias: extensionsmock${1}
+        - pkg: github.com/gardener/gardener/extensions/pkg/apis/config
+          alias: extensionsconfig
```

Comment:
We could remove this. See https://github.com/gardener/gardener/pull/12618 and https://github.com/gardener/gardener/pull/12618/commits/4df418207bb836f0929f164e8a2ec0f6513a8100.

```suggestion
```